css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,129 @@
+$sc-padding-sm: 0.5em 1em !default;
+$sc-padding: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-border-radius-rounded: $border-radius-rounded;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-3;
+	grid-template-columns: max-content auto max-content;
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	.segmented-control-previous,
+	.segmented-control-next {
+		display: flex;
+		align-items: center;
+		padding: $sc-padding-sm;
+		border: $sc-border;
+		border-radius: $sc-border-radius-rounded;
+		background-color: $sc-active-background-color;
+		color: $sc-active-color;
+		font-weight: $weight-semibold;
+		cursor: pointer;
+
+		&:hover {
+			border-color: $sc-active-hover-border-color;
+			background-color: $sc-active-hover-background-color;
+
+			@include forced-colors {
```

Comment:
In forced color mode, next/prev buttons on hover have white text color and not accessible

![image](https://github.com/microsoft/atlas-design/assets/57374379/f0ac3866-ce6c-49c9-a801-4950b6a6774f)

Reply:
And hover on selected segment isn't accessible either:

![image](https://github.com/microsoft/atlas-design/assets/57374379/9f897e19-c38d-43eb-bbea-07118daba6d0)

Reply:
@ollips In the second screenshot, is that `#0065B3` from our colors, or is that your OS's selected color for `Highlight`?

Reply:
It's our color

![image](https://github.com/microsoft/atlas-design/assets/57374379/4828ed63-703e-4c6b-ba72-37ffb85fcc99)

Reply:
@AaronAstle I was able to fix the above issues by converting all instances of `forced-color-adjust: preserve-parent-color` in this file to `forced-color-adjust: none`.

Reply:
So long as we actually are pulling appropriate combinations from the user's system colors (such as `Highlight` and `HighlightText`) when forced colors mode is active, we're not responsible for color contrast, since it's totally valid for users to choose a low-contrast theme for their own needs/preferences.

It's when our own colors come through, like Olga showed above, that we have an issue.

Reply:
adjusted. ✅

Reply:
Seems to work mostly as expected, though I do notice that our brand blue is still bleeding through when forced colors is active and I hover over the selected tab.

I'm thinking maybe we should retool the system colors like this: 

- Selected tab:
  - Background color: `SelectedItem`
  - Text color: `SelectedItemText`
- Hovered tab _(ideally, this would take precedence over the selected tab's styles)_:
  - Background color: `Highlight`
  - Text color: `HighlightText`

Reply:
@AaronAstle and I paired to get the above implementation in - it works well, from what testing we've got.

@ollips, are you fine with resolving this thread?